Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -149,6 +156,10 @@ public void StartKeywordRecognizer()
             {
                 keywordRecognizer.Start();
             }
+            else
```

Comment:
The logic here seems a bit off. If someone calls `StartKeywordRecognizer` twice, keyBindingsOnly will be set to true.